app/components/gcse_requirements_component.rb

```diff
@@ -0,0 +1,50 @@
+class GcseRequirementsComponent < ViewComponent::Base
+  attr_reader :course
+
+  def initialize(course:)
+    @course = course
+  end
+
+private
+
+  def required_gcse_content(course)
+    case course.level
+    when "primary"
+      "Grade 4 (C) or above in English, maths and science, or equivalent qualification"
+    when "secondary"
+      "Grade 4 (C) or above in English and maths, or equivalent qualification"
+    end
+  end
+
+  def pending_gcse_content(course)
+    if course.accept_pending_gcse.present?
+      "Candidates with pending GCSEs will be considered"
+    else
+      "Candidates with pending GCSEs will not be considered"
+    end
+  end
+
+  def gcse_equivalency_content(course)
+    return if equivalencies.count.zero? && course.additional_gcse_equivalencies
```

Comment:
Wondering if this would be slightly easier to follow if we leaned on `accept_gcse_equivalency?`, eg—

```ruby
    return "Equivalency tests will not be accepted" unless course.accept_gcse_equivalency?

    case equivalencies.count
    when 0
      "" # Assume that course.additional_equivalencies is populated instead
    when 1
      "Equivalency tests will be accepted in #{equivalencies[0].capitalize}"
    when 2
      "Equivalency tests will be accepted in #{equivalencies[0].capitalize} and #{equivalencies[1]}"
    when 3
      "Equivalency tests will be accepted in #{equivalencies[0].capitalize}, #{equivalencies[1]} and #{equivalencies[2]}"
    end
```